### ISSUE TITLE
[BUG] Accessibility - Web App - Page language has not been identified.

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5183]
**Describe the bug**
Page language has not been identified. The main language o the page has not been declared within the HTML element. Language tags tell a screen reader how to pronounce the text on the page. These missing or being mistyped could make screen reader revert to default settings and result in a strange user experience.
[WCAG 3.1.1 Language of Page (Primary)](https://www.w3.org/TR/WCAG21/#language-of-page)

**To Reproduce**
1. Go to Firefox browser
2.Navigate to app.illinois.edu
3. Run AInspector extension

**Expected behavior**
A page language should be programatically determined.

**Screenshots**

![Image](https://github.com/user-attachments/assets/cdca1e10-fcb6-4161-b09e-a787cc8ab163)

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Smartphone (please complete the following information):**

**Additional context**

**Comment by dobromirdobrev** [comment]
Hi @acovher2 ,

This is fixed in version 7.3.18 of the Illinois web app.

**Comment by dobromirdobrev** [comment]
Hi @acovher2 ,

The new version 7.3.18 has just been deployed.

And yes, in future, I'll assign the ticketets once the version is available.
